src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -635,7 +639,9 @@ template <typename I>
 void ImageReplayer<I>::handle_start_replay(int r) {
   dout(10) << "r=" << r << dendl;
 
-  if (r < 0) {
+  if (on_start_interrupted()) {
+    return;
+  } if (r < 0) {
```

Comment:
This happens to work because of `return`, but better to change to `else if`.

Reply:
Ah. Sure! Thanks!